index.html

```diff
@@ -16,11 +16,18 @@
   <header>
     <div class="welcome-container">
       <a href="/">
-        <img class="all-image" src="images/Welcome.svg" alt="Welcome">
+        <img class="all-image d-none" src="images/Welcome.svg" alt="Welcome">
       </a>
     </div>
+    <div class="desktop-nav">
+      <ul class="nav-link">
+        <li><a class="nav-links" href="">Portfolio</a></li>
+        <li><a class="nav-links" href="">About</a></li>
+        <li><a class="nav-links" href="">Contact</a></li>
+      </ul>
+    </div>
     <nav>
-      <div class="image-container">
+      <div class="image-container d-none">
         <img class="all-image" src="images/Menu.svg" alt="Menu icon">
       </div>
     </nav>
```

Comment:
The navigation menu for mobile view shows a list of the menu which is not in the design. The mobile menu should only show when the user clicks on the hamburger button. Kindly see the below screenshot and fix the navigation accordingly.
![navigation-mobile](https://user-images.githubusercontent.com/35504149/131216438-96b2c160-f39c-468f-a230-7bdd18d212ec.png)